packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_profile/views/profile_page.dart

```diff
@@ -37,6 +39,19 @@ class ProfilePage extends StatelessWidget {
                     .push(const NotificationsRoute()),
               ),
             ),
+            SizedBox(
+              height: context.designSystem.spacing.xl0,
+            ),{{#enable_internationalisation}}
+            LanguagePickerButton(
+              service: context.read<LanguageServiceExample>(),
+              onChanged: (language) => context
+                  .read<CoordinatorBlocType>()
+                  .events
+                  .setCurrentLanguage(language),
+              padding: context.designSystem.spacing.xl0,
+              buttonText: context.l10n.changeLanguage,
+              translate: (model) => '',
+            ),{{/enable_internationalisation}}
```

Comment:
Same as above

Reply:
This var was changed from `enable_internationalisation ` to `enable_change_language`